include/enclave/crypto.h

```diff
@@ -245,6 +245,9 @@ static int verifySignature(mbedtls_pk_context pk, uint8_t* data, size_t data_len
   }
 
   if((ret = mbedtls_pk_verify(&pk, MBEDTLS_MD_SHA256, hash, 0, signature, sig_len)) != 0 ) {
+    // char error_buf[200];
```

Comment:
Cleanup commented code? On a related note, would it be more descriptive for the LOG to be outputting the results of `mbedtls_strerror`?

Reply:
Yeah, I was thinking the same thing. Added an issue in Jira to track this.